### PR TITLE
regenerate sdk

### PR DESCRIPTION
**Pull request by dirtycajunrice**
## Changes

- Linear: https://linear.app/thirdweb/issue/INF-171/
- Here is the problem being solved.
- Here are the changes made.

## How this PR will be tested

- [ ] Open the dashboard and click X. Result: A modal should appear.
- [ ] Call the /foo/bar API. Result: Returns 200 with "baz" in the response body.

## Output

(Example: Screenshot/GIF for UI changes, cURL output for API changes)

### CODE REVIEW

**Comment by dirtycajunrice** [comment]
* **#808** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/engine/808?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thirdweb-dev/engine/808?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (arcoraven) [inline .idea/.gitignore:1]: Can you add .idea to the root .gitignore so this folder doesn't get checked in?

**Review by arcoraven — COMMENTED** [review]
I'm not sure why every file's whitespace got changed. Is this intended or expected?

File: src/server/middleware/auth.ts [hunk 2]
```diff
@@ -123,7 +123,7 @@ export async function withAuth(server: FastifyInstance) {
         }
         // Allow this request to proceed.
         return;
-      } else if (error) {
+      }if (error) {
```
Comment (arcoraven) [inline src/server/middleware/auth.ts:126]: These should be newlines

**Review by arcoraven — APPROVED** [review]

**Pull request merged**